File: Regional 2019/B.-PoniendoLaMesa/B.java
import java.util.Scanner;

public class PoniendoLaMesa {

	public static void main(String[] args) {

		Scanner Scan = new Scanner(System.in);
        
		while(true) {
			 			
			int copas=Scan.nextInt();
			
			if(copas==0) break;
		
			if(copas==1)System.out.println(1);
            		else System.out.println(copas-(copas%2));
		}
	}
}

File: Regional 2019/B.-PoniendoLaMesa/B.c
#include <stdio.h>
int main(void){
    
    int copas;
    while(1){
        scanf("%i", &copas);
        if (copas==0)break;
        
        if(copas!=1){
            printf ("%d\n",copas-copas%2); 
       }else{
            printf ("1\n");
        }
    }
}
